extensions/kubeapi/rbac/list.go

```diff
@@ -165,3 +165,28 @@ func ListRoleTemplates(client *rancher.Client, listOpt metav1.ListOptions) (*v3.
 
 	return rtList, nil
 }
+
+// ListProjectRoleTemplateBindings is a helper function that uses the dynamic client to list projectroletemplatebindings from local cluster.
+func ListProjectRoleTemplateBindings(client *rancher.Client, listOpt metav1.ListOptions) (*v3.ClusterRoleTemplateBindingList, error) {
```

Comment:
Shouldn't this function return `v3.ProjectRoleTemplateBindingList` instead of `v3.ClusterRoleTemplateBindingList`?